frontend/src/components/help-panel/TitleDescriptionHelpPanel.tsx

```diff
@@ -34,7 +34,11 @@ function TitleDescriptionHelpPanel({
           </h3>
           <ul>
             <li>
-              <a href={t('helpPanel.footer.docs.link')}>
+              <a
```

Comment:
suggestion I think you can use [Link external by Cloudscape](https://cloudscape.design/components/link/?tabId=playground&example=external-link) here

Reply:
Thanks, will fix in next revision